go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/alecthomas/kong v0.9.0
 	github.com/go-resty/resty/v2 v2.11.0
 	github.com/google/uuid v1.6.0
+	github.com/knqyf263/go-deb-version v0.0.0-20230223133812-3ed183d23422
```

Comment:
Whole library is ~300 lines of code and not really popular. I think it will be better just to write our own implementation of required functions and do not have dependency on something that we not sure that will be well maintained.

Reply:
not sure we need to maintain already implemented functionality

Reply:
We will ship this lib on thousands servers. My concerns related to the fact the Percona probably is the biggest user of this lib, and in case of some compromise we won't notice this. We can be sure in our code, but we can't be sure in the lib with 34 stars.